pyfolio/plotting.py

```diff
@@ -29,6 +29,8 @@
 from . import timeseries
 from . import pos
 
+from .utils import BDAYS_PER_MONTH
```

Comment:
We could also overload the variable and do a typecheck in the function

WARNING: Syntax is approximate

```python
if type(rolling_window) is datetime.timedelta:
    num_days = rolling_window.days
elif type(rolling_window) is Number:
    num_days = rolling_window
else:
    raise TypeError